packages/mgt-components/src/graph/graph.presence.ts

```diff
@@ -71,7 +71,7 @@ export const getUserPresence = async (graph: IGraph, userId?: string): Promise<P
  * @returns {}
  * @memberof BetaGraph
  */
-export const getUsersPresenceByPeople = async (graph: IGraph, people?: IDynamicPerson[]) => {
+export const getUsersPresenceByPeople = async (graph: IGraph, people?: IDynamicPerson[], allowCache = true) => {
```

Comment:
I'm not a huge fan of how the allowCache parameter works in the current implementation.
I'd like to see the cache updated when a presence is fetched that way, we still have a nice cache aside pattern working here.

IMO it might be better called `bypassCacheRead` or something of that nature.